camera_lucidTypes.hpp

```diff
@@ -106,6 +112,22 @@ namespace camera_lucid {
         int offset_y = 0;
     };
 
+    struct CameraConfig {
+        /** Camera's ip*/
+        std::string ip;
+        /** Timeout after factory reset*/
+        base::Time camera_reset_timeout = base::Time::fromSeconds(50);
+        /** Period of info update (such as temperature, pressure...)*/
+        base::Time update_info = base::Time::fromSeconds(1);
+        /** Choose the temperature sensor*/
+        DeviceTemperatureSelector temperature_selector =
+            DeviceTemperatureSelector::DEVICE_TEMPERATURE_SELECTOR_SENSOR;
+    };
+
+    struct CameraInfo {
+        /** Camera's temperature from Sensor mode*/
+        float temperature = 0;
```

Comment:
There's a base/Temperature type. Moreover, float values should be initialized as `base::unknown<float>()` or `base::unknown<double>()` (which is semantic sugar on top of NaN)